identifiers/country-fr/oversea.csv

```diff
@@ -0,0 +1,12 @@
+id,name
+ocd-division/country:fr/oversea_collectivity:mf,St-Barthélemy/St-Martin
+ocd-division/country:fr/oversea_collectivity:nc,Nouvelle-Calédonie
+ocd-division/country:fr/oversea_collectivity:pf,Polynésie français
+ocd-division/country:fr/oversea_collectivity:pm,St-Pierre-et-Miquelon
+ocd-division/country:fr/oversea_collectivity:wf,Wallis-et-Futuna
+ocd-division/country:fr/oversea_department:français_etablis_hors_de_france,Français établis hors de France
```

Comment:
I'm curious about this oversea_department, does it exist solely to be a parent of the 11 constituencies for French residents over seas? https://en.wikipedia.org/wiki/Constituencies_for_French_residents_overseas

It sounds like these constituencies are directly by the French government. If my understanding is correct, then I don't think an explicit OCD ID should exist for this.

Reply:
Yes, it represents the 11 constituencies. 
A note from wikipedia : each one of this constituency elect one member of the National Assembly + senators representing French expatriates. 

I don't get your last comment, can you please clarify ?

Reply:
Hi @jloutsenhizer.

what you descirbe applies only to last const. 
(ocd-division/country:fr/oversea_department:français_etablis_hors_de_france,Français établis hors de France)
for all french people, who live abroad. So we need this ocd-id to flag that a person takes part 
We have the same const. also in croatia and one or two other countries.

This const. has nothing to do with the other "oversea_collectivity".
BUT, for senates election, this ocd was not relevant.
So we could exclude this ocd this time and talk about it next election? @azuser @jloutsenhizer

Reply:
Now with the lower house districts in this looks more relevant. Given that something similar is already being being used I think this is good as is.

Reply:
This is not an `oversea_department`: https://en.wikipedia.org/wiki/Overseas_department_and_region.

Use `ocd-division/country:fr/special:diaspora` like for Croatia.